classes/class-p4ct-site.php

```diff
@@ -324,8 +324,10 @@ public function enqueue_public_assets( $hook ) {
 		wp_enqueue_style( 'child-style-fonts', get_stylesheet_directory_uri() . '/static/css/' . $css_fonts, [], $css_creation );
 
 		wp_enqueue_style( 'child-style-swiper', 'https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.5.0/css/swiper.min.css' );
+    wp_enqueue_style( 'child-style-flexboxgrid', 'https://cdnjs.cloudflare.com/ajax/libs/flexboxgrid/6.3.1/flexboxgrid.min.css' );
```

Comment:
Please indent with tabs (or spaces) consistent cross the files.

Reply:
Can we move this style into the webpack bundles? Instead of adding many includes here.